styles/shared/spacing.css

```diff
@@ -0,0 +1,198 @@
+/**********************************************************************************************************/
+/***************************************************** MARGINS ********************************************/
+
+.ml-auto {
+    margin-left: auto;
+}
+.mr-auto {
+    margin-right: auto;
+}
+.mx-auto {
+    margin-left: auto;
+    margin-right: auto;
+}
+.mx-25 {
+    margin-left: 25%;
+    margin-right: 25%;
+}
+.mx-50 {
+    margin-left: 50%;
+    margin-right: 50%;
+}
+
+.mt-auto {
+    margin-top: auto;
+}
+.mb-auto {
+    margin-bottom: auto;
+}
+.my-auto {
+    margin-top: auto;
+    margin-bottom: auto;
+}
+.my-25 {
+    margin-top: 25%;
+    margin-bottom: 25%;
+}
+.my-50 {
+    margin-top: 50%;
+    margin-bottom: 50%;
+}
```

Comment:
you can use margin: 50% 0;
Try to make code less repetitive and concise.

Reply:
@shrey27 here the case is the same as above.